backend/director/agents/web_search_agent.py

```
@@ -0,0 +1,172 @@
+        elif output_type == "images":
+            return self._handle_image_search(query, count)
+        else:
+            logger.error(f"Unsupported output type: {output_type}")
+            return AgentResponse(
+                status=AgentStatus.ERROR,
+                message=f"Unsupported output type: {output_type}",
+            )
+
+    def _handle_video_search(self, query: str, count: int, duration: str, collection_id: str) -> AgentResponse:
+        """Handles video searches."""
+        videos_content = VideosContent(
```

Ruff (0.8.2)

Undefined name `VideosContent`

(F821)